tests/integ/test_lineage.py

```diff
@@ -247,6 +246,33 @@ def test_lineage_trace(session):
         "DISTANCE": [1, 2],
     }
 
+    expected_df = pd.DataFrame(expected_data)
+    # TODO Enable the check after SNOW-1437475 is fixed.
+    # assert_frame_equal(df, expected_df, check_dtype=False)
+
+    # CASE 7 : Column lineage
+    df = session.lineage.trace(
+        f"{db}.{schema}.V2.C1", "COLUMN", direction=LineageDirection.UPSTREAM
+    )
+
+    # Removing 'creadtedOn' field since the value can not be predicted.
+    df = remove_created_on_field(df.to_pandas())
+
+    expected_data = {
+        "SOURCE_OBJECT": [
+            {"domain": "COLUMN", "name": f"{db}.{schema}.V1.C1", "status": "ACTIVE"},
+            {"domain": "COLUMN", "name": f"{db}.{schema}.T1.C1", "status": "ACTIVE"},
```

Comment:
do we need to include a new filed in the output for column's parent table domain? so we can indicate if it is view, featureView, etc. DGQL result already has the `tableType` field in `properties`

Reply:
Good point. Exposing it as filed name "type" to consistent with srcType and dstType in SQL API